Token_Contracts/test/humanStandardToken.js

```diff
@@ -130,6 +130,21 @@ contract("HumanStandardToken", function(accounts) {
         }).catch(done);
```

Comment:
Through Tim's help, I grok it now. catch(done) is still required since the tests need to continue in the event of an error. However, it seems done is never needed, so might have to revamp the whole test suite. It works for now, so not a priority.